patterns/hero-full-width-image.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Hero Full Width Image
+ * Slug: twentytwentyfive/hero-full-width-image
+ * Categories: banner, featured
+ * Description: A hero with a full width image, heading, short paragraph and button
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-8802603-original.webp","dimRatio":20,"overlayColor":"black","isUserOverlayColor":true,"focalPoint":{"x":0.5,"y":0.95},"minHeight":832,"minHeightUnit":"px","contentPosition":"bottom left","align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull has-custom-content-position is-position-bottom-left" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50);min-height:832px"><span aria-hidden="true" class="wp-block-cover__background has-black-background-color has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php echo esc_attr_x( 'Picture of a flower', 'Alt text for cover image', 'twentytwentyfive' ); ?>" src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-8802603-original.webp" style="object-position:50% 95%" data-object-fit="cover" data-object-position="50% 95%"/>
+	<div class="wp-block-cover__inner-container">
```

Comment:
My interpretation of the design is that the content of the cover (title, paragraph, buttons) should be limited to wide width. 

This was also mentioned somehow [here](https://github.com/WordPress/twentytwentyfive/pull/129#issuecomment-2312171127).

Could you please modify the pattern so that the contents are limited site-wide but the cover background is full width?

Reply:
Hi @juanfra 

I believe you are correct here, thanks for pointing that out.

To solve, I've had to add a group block containing the Heading, Paragraph and Button block which is aligned to the wide width (1340px), but then restrained the **content width** within to 718px as per the design.

<img width="1707" alt="Screenshot 2024-09-02 at 15 18 36" src="https://github.com/user-attachments/assets/b719fef1-edb9-488c-9bab-4a1b8c36d9c5">